integration/dbt/openlineage/dbt/__init__.py

```diff
@@ -308,6 +309,7 @@ def consume_local_artifacts(
                 exc_info=True,
             )
     logger.info("Emitted %d OpenLineage events", emitted_events)
+    logger.info("Underlying dbt execution returned %d", return_code)
```

Comment:
any reason for not having a single log message?

Reply:
Those messages are about different things. I don't think having them separate can cause any issues, and is more readable.